content/integrate/redis-data-integration/data-pipelines/deploy.md

```diff
@@ -54,6 +50,88 @@ secrets are only relevant to TLS/mTLS connections.
 | `TARGET_DB_CERT` | (For mTLS only) Target database public key |
 | `TARGET_DB_KEY_PASSWORD` | (For mTLS only) Target database private key password |
 
+### Set secrets for VM deployment
+
+Use
+[`redis-di set-secret`]({{< relref "/integrate/redis-data-integration/reference/cli/redis-di-set-secret" >}})
+to set secrets for a VM deployment. For example, you would use the
+following command line to set the source database username to `myUserName`:
+
+```bash
+redis-di set-secret SOURCE_DB_USERNAME myUserName
+```
+
+### Set secrets for K8s/Helm deployment
+
+Use
+[`kubectl create secret generic`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret_generic/)
+to set secrets for a K8s/Helm deployment. The general pattern of the commands is
+
+```bash
+kubectl create secret generic <DB> \
+--namespace=rdi \
+--from-literal=<SECRET-NAME>=<SECRET-VALUE>
+```
+
+Where `<DB>` is either `source-db` for source secrets or `target-db` for target secrets.
+The specific command lines for source secrets are as follows:
+
+```bash
+# Source username
+kubectl create secret generic source-db \
+--namespace=rdi \
+--from-literal=SOURCE_DB_USERNAME=yourUsername
+
+# Source password
+kubectl create secret generic source-db \
+--namespace=rdi \
+--from-literal=SOURCE_DB_PASSWORD=yourPassword
+
+# Source trust certificate
+kubectl create secret generic source-db \
+--namespace=rdi \
+--from-literal=SOURCE_DB_CACERT=/etc/certificates/source_db/ca.crt
```

Comment:
this is incorrect. There are two commands for each certificate otherwise it will fail. One loads the file from the local folder and the other references it (in the future we will try to remove the latter, but for now it is needed).
e.g.:
```
kubectl create secret generic target-db-ssl --from-file=ca.crt=/path/to/myca.crt -n rdi
kubectl create secret generic target-db \
--namespace=rdi \
--from-literal=TARGET_DB_CACERT=/etc/certificates/target-db/ca.crt
```